module/Photo/src/Photo/Controller/PhotoController.php

```diff
@@ -10,17 +10,21 @@ class PhotoController extends AbstractActionController
 
     public function indexAction()
     {
-        $albums = $this->getAlbumService()->getAlbums();
         //add any other special behavior which is required for the main photo page here later
-
+        $years = $this->getAlbumService()->getAlbumYears();
+        $year = $this->params()->fromRoute('year');
+        // If no year is supplied, use the current year.
+        if(is_null($year)) $year = (date("m") < 7) ? date("Y") - 1 : date("Y");
```

Comment:
$year = max($years) is beter want:

1. Je hoeft er niet van uit meer te gaan dat $years gesorteerd is
2. end($years) heeft side effects (De interne pointer wordt naar het enide gezet)